Makefile

```diff
@@ -7,6 +7,8 @@ generate: tidy
 
 # Coding style static check.
 lint: tidy
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.56.2
```

Comment:
I wouldn't put this in the Makefile target, as this would have side-effects every time somebody calls `make lint` on his local machine.

Reply:
It is fixed now !